variables.tf

```diff
@@ -79,3 +79,62 @@ variable "server_name" {
   description = "The name of the server"
   default     = "default"
 }
+
+variable "channel" {
+  type = string
+  description = "The version of channel"
+  default = "v1.9"
+}
+
+variable "storage_class" {
+  type = string
+  description = "The provisioned block or file storage class for all the PVCs required by Cloud Pak for Security. When it is not specified, the default storage class is used."
+  default = ""
+}
+
+variable "roks_auth" {
+  type = string
+  description = "Whether ROKS authentication need to be enabled"
+  default = "false"
+}
+
+variable "entitlement_key" {
+  type = string
+  description = "Entitlement key value"
+}
+
+variable "admin_user" {
+  type = string
+  description = "Short name or email-id of the user to be given administrator privileges in the default account. Mandatory value while creating cp4s-threat-management-instance"
+  default = "admin"
+}
+
+variable "catalog" {
+  type        = string
+  description = "The catalog source that should be used to deploy the operator"
+  default     = "ibm-operator-catalog"
+}
+
+variable "catalog_namespace" {
+  type        = string
+  description = "The namespace where the catalog has been deployed"
+  default     = "openshift-marketplace"
+}
+
+variable "backup_storage_size" {
+  type = string
+  description = "The storage size for the backup and restore PVC. Must be 500Gi or higher."
+  default = ""
+}
+
+variable "backup_storage_class" {
```

Comment:
What is the default storage class that gets set here?   Those 500Gi portworx claims are going to add up quickly for our testing, and eat up available portworx storage (if it is using portworx).  For the test case, can we use something like `ibmc-vpc-block-general-purpose`?

Reply:
@triceam I have left it blank to use default storage class which is patched in cluster i.e `ibmc-block-gold`. However, after knowing CP4S is creating resource crunch by consuming lots of PVCs I am deleting the instance after making sure it is deployed successfully (As soon as destroy completes in workflow and deletes the test repository) and also uninstalling the CP4s operator, so that it will not continue and reserve those PVCs.